edgelet/contrib/centos/aziot-edge.spec

```diff
@@ -17,7 +17,7 @@ URL:            https://github.com/azure/iotedge
 %{?systemd_requires}
 BuildRequires:  systemd
 Requires(pre):  shadow-utils
-Requires:       aziot-identity-service >= 1.2.0-1
+Requires:       aziot-identity-service >= @version@-@release@
```

Comment:
This is going to break internal builds. That's why it was changed to the explicit versions in the first place.